src/pybind/mgr/cephadm/module.py

```diff
@@ -818,6 +819,7 @@ def _get_user(self) -> Tuple[int, str, str]:
         """
         Show user for SSHing to cluster hosts
         """
+        assert self.ssh_user
```

Comment:
this might be False. Returning `None` would be better imo

Reply:
@sebastian-philipp how about
```py
return -errno.ENOENT, '', 'No cluster SSH user configured'
```
?

Reply:
updated in a separate commit.